### DISCUSSION TITLE
Silly but why not use ChatGPT

### DISCUSSION THREAD

**Discussion by abinshac** [official-stockfish/Stockfish#4433] [Ideas]
I just tried asking chatgpt if there‘s any way stockfish 15 can be improved. It gave me a 100 suggestions and I am not really a programmer but I think it could help. I can give some screenshots as example. Please don't criticize me. I admit that I don't know much about programming but I thought this might needed your attention. I might be wrong too.

![2031FB45-03B9-47A7-9649-B2D09B89695B](https://user-images.githubusercontent.com/67497836/223945001-87524f29-ae19-4e8d-8ea1-16299065a000.jpeg)
 
![C649C1DC-528B-4EE9-995F-A39F1D695D79](https://user-images.githubusercontent.com/67497836/223945009-e00b5a1b-8c46-4560-bc76-87b36cf4f04f.jpeg)

![22A11EFC-36DB-4379-9B72-CE18F50191E6](https://user-images.githubusercontent.com/67497836/223944921-5ce4c4d2-211f-4418-9abe-bd233ddc01f3.jpeg)

![CF2752F1-2126-4E21-B82B-9FB42CC9C556](https://user-images.githubusercontent.com/67497836/223944970-90b7b7aa-3b78-41ad-851d-b5c19fe6bd68.jpeg)

![D8409038-1947-4D86-A785-3D1234B23FCC](https://user-images.githubusercontent.com/67497836/223944978-bd892974-51b8-4912-9ddc-8b17d3666b0c.jpeg)

![B3F8120B-5495-42F1-971C-E5B28A600C76](https://user-images.githubusercontent.com/67497836/223944997-1bbc0dc5-e9c4-4985-91bc-ba259c6dd5fc.jpeg)

**Comment by Disservin** [comment]
None of these advices are really helpful. Most of the stuff already exists in some way and chatgpts knowledge about chess engine programming comes from online resources about engine programming. Coming up with something entirely new which is also logical is not really chatgpts strength

**Comment by winapiadmin** [comment]
that is... pseudocode. Also don't vibe coding. They are already exist in the codebase.
